cmd/crates/soroban-spec-typescript/ts-tests/src/util.ts

```diff
@@ -0,0 +1,27 @@
+import { Keypair, TransactionBuilder } from "soroban-client";
+
+export const rpcUrl = process.env.SOROBAN_RPC_URL ?? "http://localhost:8000/";
+export const secretKey =
+  "SC36BWNUOCZAO7DMEJNNKFV6BOTPJP7IG5PSHLUOLT6DZFRU3D3XGIXW";
+
+const keypair = Keypair.fromSecret(secretKey);
```

Comment:
should/can this just be `Keypair.random()`?

Reply:
It could be but it was easier this way as it is the same key as `soroban config identity show --hd-path 0`, which is funded.

Reply:
makes sense, nice!